package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/web-reader",
-  "version": "0.3.3",
+  "version": "1.0.0",
```

Comment:
I think we should stay below 1.0.0 until the remove R2D2BC PR is merged. It doesn't actually break any APIs or features, but it's a pretty major overhaul and I think it'd be good to wait on the 1.0. What do you think?

Reply:
We talked about this in DRB, and we can all get on a call and talk about this - but the web reader should have been in V1 when it released in October.  If the R2D2BC stuff causes breaking changes, it should be V2.  This kind of makes sense - the web reader has been in production, stably, for some months.  Really it's semantic and if you feel strongly, that's fine.  We were just getting questions as to "when is web reader released" and the answer that felt truest to me was that it was "released" in October but we forgot to bump the number then.